Xamarin.PropertyEditing.Mac/Controls/EnumEditorControl.cs

```diff
@@ -0,0 +1,83 @@
+using System;
+using System.Collections;
+using System.ComponentModel;
+using System.Diagnostics;
+using AppKit;
+using CoreGraphics;
+using Foundation;
+using Xamarin.PropertyEditing.ViewModels;
+
+namespace Xamarin.PropertyEditing.Mac
+{
+	internal class EnumEditorControl : PropertyEditorControl
+	{
+		NSComboBox ComboBoxEditor;
+
+		dynamic EnumEditorViewModel => ViewModel;
```

Comment:
@ermau Why? Won't there be equal overheads whether the DLR finds the right object or whether we test for it and create the appropriate object. This way the code is much *much*  simpler to read.